### PR TITLE
embedding homepage v2

### PR DESCRIPTION
**Pull request by npretto**
This implements the new UI from [here](https://www.figma.com/design/W1TfzdrR8ldzp4vdjP1z36/SDK-Embedding%3A-admin-settings-%2B-embed-flow?node-id=11346-19786&t=Xp8nFfkMzoEdXq6n-4).

We're trying to ship this on the gold release so I may do a follow up PR to clean up un-used settings and code

### CODE REVIEW

File: e2e/test/scenarios/onboarding/embedding-homepage.cy.spec.ts [hunk 1]
```diff
@@ -29,37 +28,14 @@ describeWithSnowplow("scenarios > embedding-homepage > snowplow events", () => {
     expectNoBadSnowplowEvents();
   });
 
-  it("clicking on the quickstart button should send the 'embedding_homepage_quickstart_click' event", () => {
```
Comment (npretto) [inline e2e/test/scenarios/onboarding/embedding-homepage.cy.spec.ts:32]: I'm waiting for alberto to confirm, but I don't think we need this event anymore, not in this shape at least

File: frontend/src/metabase/home/components/EmbedHomepage/NeedMoreInfoCard.tsx [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+  embeddingDocsUrl,
+  analyticsDocsUrl,
+}: NeedMoreInfoCardProps) => (
+  <Card px="xl">
+    <Text
+      lh={1.57}
+      color="text-dark"
+      fw="bold"
+    >{t`Need more information?`}</Text>
+    <Text lh={1.57} color="text-light" size="sm">
```
Comment (oisincoveney) [inline frontend/src/metabase/home/components/EmbedHomepage/NeedMoreInfoCard.tsx:21]: how was 1.57 calculated?

File: frontend/src/metabase/home/components/EmbedHomepage/StaticEmbedContent.tsx [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+      color="text-medium"
+    >{t`Static embedding`}</Text>
+    <Text lh={1.57} mb="md">
+      {/* eslint-disable-next-line no-literal-metabase-strings -- This string only shows for admins. */}
+      {t`Embed a dashboard in a 'Powered by Metabase' iframe with interactivity limited to filters and tooltips, and a few customization options. The iframe loads a Metabase URL secured with a signed JSON Web Token (JWT). Appears with "Powered by Metabase", on Open Source and Starter plans, with the option to remove on Pro and Enterprise. As the simplest form of embedding, you can add a dashboard into your app in a few minutes with just a snippet.`}
+    </Text>
+    <StaticEmbedImage
+      src="/app/assets/img/static-embedding-example.png"
+      alt="Static embedding example"
+      // style={{ width: "100%", marginBottom: "1rem" }}
```
Comment (oisincoveney) [inline frontend/src/metabase/home/components/EmbedHomepage/StaticEmbedContent.tsx:34]: remove this thing

File: frontend/src/metabase/home/components/EmbedHomepage/StaticEmbedContent.tsx [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+        </Link>
+      )}
+      <ExternalLink href={learnMoreStaticEmbedUrl}>
+        <Button variant="subtle">{t`Read the docs`}</Button>
+      </ExternalLink>
+    </Group>
+  </Box>
+);
+
+const StaticEmbedImage = styled.img`
```
Comment (oisincoveney) [inline frontend/src/metabase/home/components/EmbedHomepage/StaticEmbedContent.tsx:52]: can we CSS this really quickly? or at least let's move this to CSS soon?

**Review by oisincoveney — APPROVED** [review]
a few questions but nothing blocking

File: frontend/src/metabase/home/components/EmbedHomepage/StaticEmbedContent.tsx (hunk 3 above)
Comment (npretto) [inline frontend/src/metabase/home/components/EmbedHomepage/StaticEmbedContent.tsx:34]:
```suggestion
```

File: frontend/src/metabase/home/components/EmbedHomepage/NeedMoreInfoCard.tsx (hunk 2 above)
Comment (npretto) [reply frontend/src/metabase/home/components/EmbedHomepage/NeedMoreInfoCard.tsx:21]: `desired line height in pixels from the design / font size`
before using it I checked and there was at least another place where it was set like that

File: frontend/src/metabase/home/components/EmbedHomepage/StaticEmbedContent.tsx (hunk 4 above)
Comment (npretto) [reply frontend/src/metabase/home/components/EmbedHomepage/StaticEmbedContent.tsx:51]: I'd go with this implementation for now, the embed homepage will be moved in another page soon, let's see how that page is styled

File: frontend/src/metabase/home/components/EmbedHomepage/NeedMoreInfoCard.tsx (hunk 2 above)
Comment (npretto) [reply frontend/src/metabase/home/components/EmbedHomepage/NeedMoreInfoCard.tsx:21]: (Not a fan of the magic number, but we need to release this today)

**Pull request merged**